### PR TITLE
chore: migrate from k8s.gcr.io to registry.k8s.io

### PR DESCRIPTION
**Pull request by ahmedwaleedmalik**
Kubernetes is migrating its image registry from [k8s.gcr.io](http://k8s.gcr.io/) to [registry.k8s.io](http://registry.k8s.io/).

Part of kubernetes/k8s.io#4780.

### CODE REVIEW

**Comment by ahmedwaleedmalik** [comment]
@mrgrain can you PTAL?

File: test/fixtures/guestbook-all-in-one.yaml [hunk 1]
```diff
@@ -8,8 +8,8 @@ metadata:
     role: master
 spec:
   ports:
-  - port: 6379
-    targetPort: 6379
+    - port: 6379
```
Comment (vinayak-kukreja) [inline test/fixtures/guestbook-all-in-one.yaml]: Hey, please remove these additional spaces.

**Review by vinayak-kukreja — CHANGES_REQUESTED** [review]
Hey, thank you for your contribution. Just a minor comment.

**Comment by vinayak-kukreja** [comment]
Looks like build is failing for mutation since this was originated from a fork. Could you update your branch?

```
Run echo "::error::Files were changed during build (see build log). If this was triggered from a fork, you will need to update your branch."
  echo "::error::Files were changed during build (see build log). If this was triggered from a fork, you will need to update your branch."
  cat .repo.patch
  exit 1
  shell: /usr/bin/bash -e {0}
  env:
    CI: true
Error: Files were changed during build (see build log). If this was triggered from a fork, you will need to update your branch.
diff --git a/test/__snapshots__/yaml.test.ts.snap b/test/__snapshots__/yaml.test.ts.snap
index dd2e186..f6c89c0 100644
--- a/test/__snapshots__/yaml.test.ts.snap
+++ b/test/__snapshots__/yaml.test.ts.snap
@@ -87,7 +87,7 @@ Array [
         "spec": Object {
           "containers": Array [
             Object {
-              "image": "registry.k8s.io/redis:e2e",
+              "image": "k8s.gcr.io/redis:e2e",
               "name": "master",
               "ports": Array [
                 Object {
Error: Process completed with exit code 1.

File: test/fixtures/guestbook-all-in-one.yaml (hunk 1 above)
Comment (ahmedwaleedmalik) [reply test/fixtures/guestbook-all-in-one.yaml]: Updated, PTAL.

**Review by vinayak-kukreja — APPROVED** [review]

**Comment by vinayak-kukreja** [comment]
Hey, @ahmedwaleedmalik, our build step cannot run on a fork. Could you run `yarn build` on your fork and update here.

**Comment by ahmedwaleedmalik** [comment]
> Hey, @ahmedwaleedmalik, our build step cannot run on a fork. Could you run `yarn build` on your fork and update here.

Not sure what you want from me here? Do you want me to build it and check if it's working? Then yes, i was able to build it.

**Comment by vinayak-kukreja** [comment]
Hey @ahmedwaleedmalik, we have taken care of this issue and would be closing this PR now. 

We thank you for your contribution and bringing visibility into this. Really appreciate it. :)

**Comment by ahmedwaleedmalik** [comment]
> Hey @ahmedwaleedmalik, we have taken care of this issue and would be closing this PR now.
> 
> We thank you for your contribution and bringing visibility into this. Really appreciate it. :)

So the maintainers didn't bother to reply for a month and then just closed the PR. Seems like you folks are very "open source" friendly.

**Pull request closed without merging**